test/tests/utilities.js

```diff
@@ -272,5 +272,89 @@ describe("Zotero.Utilities", function() {
 			
 			assert.isUndefined(cslJSON.PMID, 'field labels are case-sensitive');
 		});
+		it("should parse particles in creator names", function() {
+			let creators = [
+				{
+					// No particles
+					firstName: 'John',
+					lastName: 'Smith',
+					creatorType: 'author',
+					expect: {
+						given: 'John',
+						family: 'Smith'
+					}
+				},
+				{
+					// dropping and non-dropping
+					firstName: 'Jean de',
+					lastName: 'la Fontaine',
+					creatorType: 'author',
+					expect: {
+						given: 'Jean',
+						"dropping-particle": 'de',
+						"non-dropping-particle": 'la',
+						family: 'Fontaine'
+					}
+				},
+				{
+					// only non-dropping
+					firstName: 'Vincent',
+					lastName: 'van Gogh',
+					creatorType: 'author',
+					expect: {
+						given: 'Vincent',
+						"non-dropping-particle": 'van',
+						family: 'Gogh'
+					}
+				},
+				{
+					// only dropping
+					firstName: 'Alexander von',
+					lastName: 'Humboldt',
+					creatorType: 'author',
+					expect: {
+						given: 'Alexander',
+						"dropping-particle": 'von',
+						family: 'Humboldt'
+					}
+				},
+				{
+					// institutional author
+					lastName: 'Jean de la Fontaine',
+					creatorType: 'author',
+					fieldMode: 1,
+					expect: {
+						literal: 'Jean de la Fontaine'
+					}
+				},
+				{
+					// protected last name
+					firstName: 'Jean de',
+					lastName: '"la Fontaine"',
+					creatorType: 'author',
+					expect: {
+						given: 'Jean de',
+						family: 'la Fontaine'
+					}
```

Comment:
That's ok. This isn't critical, since I don't think it makes a difference. Another thing to consider is that double-quote protection is not stripped off from first name field, so if the user expects to protect first name from parsing, it would actually just keep the quotes. Simple enough for the user to go back and fix.

@dstillman, this issue shouldn't hold back the PR merge.